kong/plugins/aws-lambda/handler.lua

```diff
@@ -20,19 +25,64 @@ function AWSLambdaHandler:new()
 end
 
 local function retrieve_parameters()
-  ngx_req_read_body()
-
   return utils.table_merge(ngx_req_get_uri_args(), public_utils.get_body_args())
 end
 
+local function retrieve_body(content_type)
+  local raw_body = ngx_req_get_body_data()
+  if raw_body == nil then return nil end
+
+  if content_type then
+    if string_find(content_type, "application/json", nil, true) then
+      local body_json, err = cjson.decode(raw_body)
+      if not body_json then
+          ngx.log(ngx.ERR, "[aws-lambda] could not encode JSON ",
+          "body for forwarded request values: ", err)
```

Comment:
style: we need to align this second string with the first argument of this function:
```lua
ngx.log(ngx.ERR, "[aws-lambda] could not encode JSON ",
        "body for forwarded request values: ", err)
```